addons/sale_pdf_quote_builder/models/quotation_document.py

```diff
@@ -46,6 +46,11 @@ class QuotationDocument(models.Model):
         compute='_compute_form_field_ids',
         store=True,
     )
+    default = fields.Boolean(
```

Comment:
`default` is a widely used field param, so it might be confusing to have it as a field name as well. IMO `add_by_default` would be a bit better.